module/wallpaper.lua

```diff
@@ -154,7 +158,15 @@ local simple_index = 0
 function setters.simple(args)
     local wallpapers = prepare_list(args)
     simple_index = (simple_index % #wallpapers) + 1
-    apply(wallpapers[simple_index], args)
+    if type(args.screen) == 'table' then
+        for _,v in ipairs(args.screens) do
```

Comment:
Pretty sure you have to check if args.screens is a table here? Doesn't make a lot of sense to check if args.screen is a table and then iterate over args.screens imo. You did that in other parts of the code too, or am I missing something

Reply:
Good catch. The check is supposed to be for `args.screen`, since we are eliminating `args.screens`. Missed changing in the iteration